scripts/docker/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Basic dependencies
+ENV HOME /opt
+RUN apt-get update && apt-get install -y apt-utils curl git git-core bsdmainutils
+
+# Install python dependencies
+ENV PYENV_ROOT $HOME/pyenv
```

Comment:
This should probably be replaced with something like this

```
RUN apt-get update && \
    apt-get install -y \
    build-essential \
    libssl-dev \
    libffi-dev \
    python3-dev \
    python3-pip \
```

Reply:
atp-get updat runs a couple lines earlier

Reply:
I meant that it could be better to install python3 and then do pip3 installs for any dependencies we need, unless there's a reason why we need to use pyenv